charts/posthog/templates/clickhouse-operator/configmap.yaml

```diff
@@ -96,14 +99,22 @@ data:
     ##
     ################################################
     
-    # ClickHouse credentials (username, password and port) to be used by operator to connect to ClickHouse instances for:
+    # ClickHouse credentials (username, password and port) to be used by operator to connect to ClickHouse instances
+    # for:
     # 1. Metrics requests
     # 2. Schema maintenance
     # 3. DROP DNS CACHE
     # User with such credentials can be specified in additional ClickHouse .xml config files,
     # located in `chUsersConfigsPath` folder
-    chUsername: clickhouse_operator
-    chPassword: clickhouse_operator_password
+    chUsername: "clickhouse_operator"
+    chPassword: "clickhouse_operator_password"
+    
+    # Location of k8s Secret with username and password to be used by operator to connect to ClickHouse instances
+    # Can be used instead of explicitly specified username and password
+    chCredentialsSecretNamespace: ""
+    chCredentialsSecretName: ""
```

Comment:
maybe we should use these instead of the above (from our script similar to how we change namespace?

Reply:
As discussed on Slack, I'm inclined to don't change anything on-the-fly coming from upstream. My take is that if we want to modify the `clickhouse-operator` we can either:

1. open a PR in their repo, get it merged and approved and pull the content here
1. fork their repo and build our own destiny